config/store.go

```diff
@@ -60,3 +61,47 @@ func NewStore(dsn string, watch bool) (Store, error) {
 
 	return NewFileStore(dsn, watch)
 }
+
+func MigrateConfig(sourceStore, destinationStore Store) error {
+	// Copy config from source to destination
+	sourceStoreConfig := sourceStore.Get()
+
+	_, err := destinationStore.Set(sourceStoreConfig)
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate config")
+	}
+
+	//get the certificate files from sourceStore
+	idpCertificateFile, err := sourceStore.GetFile(*sourceStoreConfig.SamlSettings.IdpCertificateFile)
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate config")
```

Comment:
Let's update the wrapped error messages here and below to call out more clearly what might have failed here -- e.g.
```suggestion
		return errors.Wrap(err, "failed to fetch SAML IDP certificate file")
```